test/data/err_105.py:3:1 [FURB105]: Use `print() instead of `print("")`
